packages/web-components/src/components/va-featured-content/va-featured-content.tsx

```diff
@@ -7,18 +7,53 @@ import { Component, Host, h } from '@stencil/core';
  */
 @Component({
   tag: 'va-featured-content',
-  styleUrl: 'va-featured-content.css',
+  styleUrl: 'va-featured-content.scss',
   shadow: true,
 })
 export class VaFeaturedContent {
+  /**
+   * Whether or not the component will use USWDS v3 styling.
+   */
+  @Prop() uswds?: boolean = false;
+
+  @Element() el: HTMLElement;
+
+  componentDidLoad() {
+    if (!this.uswds) {
+      return
+    }
+    // add uswds classes
+    const nodes = this.el.shadowRoot
+      .querySelectorAll('slot')
+    const headline = nodes[0];
+    const content = nodes[1];
+    
+    headline.classList.add('usa-summary-box__heading');
+    content.classList.add('usa-summary-box__text');
+  }
+
   render() {
-    return (
-      <Host>
-        <div class="feature">
-          <slot name="headline"></slot>
-          <slot />
-        </div>
-      </Host>
-    );
+    const { uswds } = this;
+    if (uswds) {
+      return (
+        <Host>
+          <div class="usa-summary-box" role="region" aria-labelledby="summary-box-key-information">
```

Comment:
The `aria-labelledby` attribute references an `id` on another element. I don't see it connected to anything here.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby

Reply:
@ataker Were you able to look into this issue?

Reply:
This was removed

Reply:
This is a feature from the USWDS example that provides an accessible name to the featured content so we might want to have that integrated. Maybe when @rsmithadhoc is able to do an a11y review, he can provide some guidance for that.

![Screenshot 2023-12-06 at 8 53 39 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/d6aaccff-31f2-40f4-bb75-552f22f859b0)